tidy3d/components/medium.py

```diff
@@ -507,6 +507,39 @@ def pole_residue(self):
             name=self.name,
         )
 
+    @classmethod
+    def from_dispersion(cls, n: float, freq: float, dn_dwvl: float = 0):
+        """Convert ``n`` and wavelength dispersion ``dn_dwvl`` values at frequency ``freq`` to
+        a single-pole :class:`Sellmeier` medium.
+
+        Parameters
+        ----------
+        n : float
+            Real part of refractive index. Must be larger than or equal to one.
+        dn_dwvl : float = 0
+            Derivative of the refractive index with wavelength (1/um). Must be negative.
+        frequency : float
+            Frequency to evaluate permittivity at (Hz).
+
+        Returns
+        -------
+        :class:`Medium`
+            medium containing the corresponding ``permittivity`` and ``conductivity``.
+        """
+
+        if dn_dwvl > 0:
+            raise ValidationError("Dispersion ``dn_dwvl`` must be smaller than zero.")
+        if n < 1:
```

Comment:
based on description of the error, should this be `n <= 1`?

Reply:
applies to one above too

Reply:
Good catch, technically the conditions are correct and the descriptions should change, but I'm gonna require dn_dwvl to be strictly smaller than zero because it could trigger an error if it's exactly zero due to numerical precision. I'll update.